homeassistant/components/light/__init__.py

```diff
@@ -369,3 +370,16 @@ def state_attributes(self):
     def supported_features(self):
         """Flag supported features."""
         return 0
+
+    @asyncio.coroutine
+    def async_added_to_hass(self):
+        """Component added, restore_state using platforms."""
+        state = get_last_state(self)
+        if state:
```

Comment:
Can you convert this to a guard clause? This example is going to be copied by a lot of other component/platform developers and thus needs to be perfect 🥇 

```python
if state is None:
    return

params = …
```